google/cloud/pubsublite/internal/stream_factory.h

```diff
@@ -0,0 +1,85 @@
+#include "absl/container/flat_hash_map.h"
+#include <google/cloud/pubsublite/v1/publisher.grpc.pb.h>
+
+namespace google {
+namespace cloud {
+namespace pubsublite_internal {
+
```

Comment:
nit: missing macros for the inline namespace, see other files for examples.

Reply:
Done.